### PR TITLE
More export convertions

### PR DESCRIPTION
**Pull request by benrudolph**
@NoahCarnahan @snopoke this is the start for saved exports. i think i'm gonna pause on this work until the rebuild of exports interface is solidified
:fish: 
cc: @gcapalbo

### CODE REVIEW

File: corehq/apps/reports/templates/reports/reportdata/excel_export_data.html [hunk 1]
```diff
@@ -1,388 +0,0 @@
-{% extends report_base|default:"reports/async/bootstrap2/default.html" %}
```
Comment (benrudolph) [inline corehq/apps/reports/templates/reports/reportdata/excel_export_data.html]: seems like this is actually still used though i can't seem to find the De-Identified reports page through the UI

Comment (benrudolph) [inline corehq/apps/reports/templates/reports/reportdata/excel_export_data.html]: based on this wiki page, i think the whole thing is dead: https://confluence.dimagi.com/display/commcarepublic/De-Identify+Data

**Comment by snopoke** [comment]
Build fail seems related:
```
======================================================================
ERROR: test_compress_offline (corehq.apps.style.tests.test_compress_command.TestDjangoCompressOffline)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/style/tests/test_compress_command.py", line 85, in test_compress_offline
    if self._is_b3(filename):
  File "/mnt/corehq/apps/style/tests/test_compress_command.py", line 54, in _is_b3
    template = parser.parse(filename)
  File "/vendor/lib/python2.7/site-packages/compressor/offline/django.py", line 96, in parse
    with io.open(template_name, mode='rb') as file:
IOError: [Errno 2] No such file or directory: 'Error parsing template /mnt/corehq/apps/reports/templates/reports/reportdata/form_deid_export.html: reports/reportdata/excel_export_data.html'
```

**Comment by benrudolph** [comment]
:white_check_mark:

**Pull request merged**